pkg/controllers/disruption/emptynodeconsolidation.go

```diff
@@ -96,6 +96,10 @@ func (c *EmptyNodeConsolidation) ComputeCommand(ctx context.Context, disruptionB
 	// Get the current representation of the proposed candidates from before the validation timeout
 	// We do this so that we can re-validate that the candidates that were computed before we made the decision are the same
 	candidatesToDelete := mapCandidates(cmd.candidates, validationCandidates)
+	if invalidatedCount := len(cmd.candidates) - len(candidatesToDelete); invalidatedCount != 0 {
```

Comment:
Isn't this the same as len(candidates) != len(candidatesToDelete)

Reply:
Resolving since this was removed, but yes. I changed it temporarily to move that math out of the log statement but was going to change it back if I kept this log line.